index.js

```diff
@@ -6,9 +6,10 @@ const clonedeep = require('lodash.clonedeep')
  *
  * @param {object} cypressConfig - The cypress config object
  * @param {object} dotEnvConfig - (optional) The dotenv config object, ref: https://www.npmjs.com/package/dotenv#config
+ * @param {string} all - (optional) Whether to return all env variables. If set to false (default), only env variables prefixed with CYPRESS_ are returned.
```

Comment:
I think this is a typo? since the 3rd parameter is a boolean and not a string

Reply:
Corrected.